Document patron code endpoints (#160)
centrapay/centrapay.github.io: src/api/transacting.md

**Optional Parameters**

| Parameter            | Description                                                                        |
|:---------------------|:-----------------------------------------------------------------------------------|
| description          | Description of the payment                                                         |
| externalReference    | Unique merchant reference for the payment request                                  |
| notifyUrl            | The URL that will receive **POST** requests from the webhook                       |
| paymentExpirySeconds | The amount of seconds until a request expires, must be an integer greater than 0   |
| terminalId           | The payment system terminal Id. Required for NZ Epay integration.                  |
| deviceId             | Physical payment system device Id                                                  |

> | patronCode           | Associate this payment request with a Patron Code. Must be attached before expiry. |

Reviewer: ```suggestion
| patronCode           | Associate this payment request with an active Patron Code. |
```
